### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stark1tty**
added wider fediverse category, fixed some spellings and added another link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,8 @@
 - [Curated Account Lists](#curated-account-lists) — find people to follow
 - [Groups](#groups) — tap into what people are saying
 - [Servers/Communities](#serverscommunities) — if you need to register for an account first, start here
-- [Additional Resources](#Additonal-Resources) — for guides and resources for navigating the platform 
+- [Additional Resources](#Additonal-Resources) — for guides and resources for navigating the platform
```
Comment (nathanlesage) [inline README.md:12]:
```suggestion
- [Additional Resources](#additional-resources) — for guides and resources for navigating the platform
```

File: README.md [hunk 2]
```diff
@@ -9,7 +9,8 @@
...
+- [Wider Fediverse Information](#Wider-Fediverse-Information) — for information and resources on other Fediverse applications
```
Comment (nathanlesage) [inline README.md:13]:
```suggestion
- [Wider Fediverse Information](#wider-fediverse-information) — for information and resources on other Fediverse applications
```

File: README.md [hunk 3]
```diff
@@ -250,14 +252,37 @@ If you are in academia and do not know where to register your account in the fed
...
 
-## Additonal Resources
+## Additional Resources
 - You can find [a comprehensive list of these and many more academic Mastodon servers on fediscience.org](https://fediscience.org/server-list.html). 
 - Visit [fedi.tips](https://fedi.tips) for a general beginners guide to Mastodon. 
 - [hcommons.org has created a quick start guide specifically for researchers in the humanities](https://hcommons.org/docs/mastodon-quick-start-guide-for-humanities-scholars/) that also includes a short list of instances that are tailored to humanities specifically.
 - Visit the [Mastodon Media List](https://docs.google.com/document/d/1TTtXbf0yDXHKXwR9jpZ4kU79mmiZT5KIM7U4tBr83hY/edit#heading=h.m7f2jiwk8u11) for a comprehensive list of Academic Journals and News Outlets.
 - [TrueSciPhi.org](https://truesciphi.org/) - Various comprehensive science, philosophy, and mathematics lists covering podcasts, Mastodon, and Twitter.
-- [OpenCheck.is](https://opencheck.is/people) - Additonal lists and a method of verification for those migrating from Twitter.
+- [OpenCheck.is](https://opencheck.is/people) - Additonal lists and a verification method for those migrating from Twitter.
```
Comment (nathanlesage) [inline README.md:261]:
```suggestion
- [OpenCheck.is](https://opencheck.is/people) - Additional lists and a verification method for those migrating from Twitter.
```

File: README.md [hunk 4]
```diff
@@ -250,14 +252,37 @@ If you are in academia and do not know where to register your account in the fed
...
+- [StreetPass](https://streetpass.social/) - A browser extension that alerts you to Mastodon verification links on a Web site.
 
+## Wider Fediverse Information
+### General
```
Comment (nathanlesage) [inline README.md:265]:
```suggestion

### General
```

File: README.md [hunk 5]
```diff
@@ -250,14 +252,37 @@ If you are in academia and do not know where to register your account in the fed
...
+- [fediverse.party](https://fediverse.party/en/miscellaneous/) - A listing of software on the Fediverse with descriptions.
+
+### Lemmy/Kbin
```
Comment (nathanlesage) [inline README.md:268]:
```suggestion
### Lemmy/Kbin

```

File: README.md [hunk 6]
```diff
@@ -250,14 +252,37 @@ If you are in academia and do not know where to register your account in the fed
...
+[Lemmy](https://join-lemmy.org/) and [Kbin](https://kbin.social/) are link aggregators on the Fediverse, similar to Reddit. They are generally compatible with each other.
+
+- [sub.rehab](https://sub.rehab/) - A listing of where Reddit servers have migrated to, mostly Lemmy/Kbin.
+
+Servers:
```
Comment (nathanlesage) [inline README.md:273]:
```suggestion
#### Servers

```

File: README.md [hunk 7]
```diff
@@ -250,14 +252,37 @@ If you are in academia and do not know where to register your account in the fed
...
+- [Clinicians Exchange](https://lem.clinicians-exchange.org/) - A Lemmy server for mental health professionals and the general public to exchange ideas related to mental health. See [here](https://www.clinicians-exchange.org/) for more info.
+- [mander.xyz](https://mander.xyz/) - A Lemmy server dedicated to the sciences.
+
+### Matrix
```
Comment (nathanlesage) [inline README.md:277]:
```suggestion
### Matrix

```

File: README.md [hunk 8]
```diff
@@ -250,14 +252,37 @@ If you are in academia and do not know where to register your account in the fed
...
+[Matrix](https://matrix.org/) is a secure messaging application similar to Discord, Slack, Telegram, and WhatsApp.
+
+- [Element](https://element.io/) - Popular mobile, web, and desktop application for using the Matrix network. 
+- [Matrix](https://doc.matrix.tu-dresden.de/en/) guide from the University of Dresden.
+
+Servers:
```
Comment (nathanlesage) [inline README.md:283]:
```suggestion
#### Servers

```

**Review by nathanlesage — COMMENTED** [review]
Fixed a few typos and formatting issues – feel free to check if you find any others :)

Thanks for the work, I think the way you integrated it is fine!

**Comment by stark1tty** [comment]
Teamwork makes the dream work. :)

File: README.md [hunk 9]
```diff
@@ -53,7 +54,7 @@ The following are curated lists of academic accounts on Mastodon by discipline.
 - [Historians of Science, Medicine, and Technology](https://docs.google.com/spreadsheets/d/1MKivV_PkVZBAKBXpBkxNFEvjvYvUyE-DQHtujsh6Gww/) (Google Spreadsheet; [Form to add yourself](https://docs.google.com/forms/d/e/1FAIpQLSfBvVQv5TNW6s__ufEllGXiCP53zZuKLzYOoaaHFU7p7CiqKQ/viewform))
 - [History and Philosophy of Science](https://oliviodare.github.io/mastodon-hps/)
 - [History](https://docs.google.com/spreadsheets/d/1zmuH0ffMpCNOEjsReMk5iiqGwiUvhO2TtrM8QKu98SE/htmlview#gid=844879137) (Google Spreadsheet)
-- [Human-Computer Interaction](https://directory.hci.social/)
+- [Human-Computer Interaction](https://directory.hci.social/) - See also: [HCI Directory Bot](https://hci.social/@directory_bot)
```
Comment (nathanlesage) [inline README.md:57]: Shouldn't we do this with other servers that have bots as well …? Just curious, we don't have to do that now!

File: README.md [hunk 10]
```diff
@@ -197,6 +198,7 @@ If you are in academia and do not know where to register your account in the fed
 - [baudigital.social](https://baudigital.social/) (Architecture, civil engineering, or urban studies; run by FID BAUdigital)
 - [Biologists.social](https://biologists.social/) (The Company of Biologists community server: cell biology, developmental biology and experimental biology)
 - [Climatescienc.es](https://climatescienc.es/explore) (Climate science and related fields)
+- [Clinicians Exchange](https://mastodon.clinicians-exchange.org/) - (For mental health professionals and the general public to exchange ideas, resources and expertise including but not limited to ethics, politics, practice building, and clinical issues; See [here](https://www.clinicians-exchange.org/) for more info.)
```
Comment (nathanlesage) [inline README.md:201]:
```suggestion
- [Clinicians Exchange](https://mastodon.clinicians-exchange.org/) (For mental health professionals and the general public to exchange ideas, resources and expertise including, but not limited to, ethics, politics, practice building, and clinical issues; [more information](https://www.clinicians-exchange.org/))
```

File: README.md [hunk 11]
```diff
@@ -250,14 +253,42 @@ If you are in academia and do not know where to register your account in the fed
...
+### Matrix
+
+[Matrix](https://matrix.org/) is a secure messaging application similar to Discord, Slack, Telegram, and WhatsApp.
+
+- [Element](https://element.io/) - Popular mobile, web, and desktop application for using the Matrix network. 
+- [Matrix](https://doc.matrix.tu-dresden.de/en/) guide from the University of Dresden.
+
+#### Servers
+
+- Archaeology: ```matrix.archaeo.social```
```
Comment (nathanlesage) [inline README.md:290]: I don't quite remember how Matrix works; aren't there links…?

**Review by nathanlesage — APPROVED** [review]
Looks good!

File: README.md (hunk 11 above)
Comment (stark1tty) [reply README.md:290]: No, they must sign up on a client application.

File: README.md (hunk 9 above)
Comment (stark1tty) [reply README.md:57]: Sure, that is probably better!

File: README.md (hunk 10 above)
Comment (nathanlesage) [reply README.md:202]: Did you now take the suggestion, or only in parts, or not at all? GH is confusing me here …

**Pull request merged**